config/deploy.rb

```diff
@@ -22,7 +22,7 @@
 # set :pty, true
 
 # Default value for :linked_files is []
-append :linked_files, 'config/application.yml'
+append :linked_files, 'config/application.yml', 'public/MP_verify_e1RFEAU2b5no6OxE.txt'
```

Comment:
MP verify file is secured (only somewhat), which having the content Tencent server given to teddies shop service account register at Tencent side, so should be not submit the git, but when deploy, you have to link such files to public folder, so Tencent server can get the file content to verify the web site is equal to what register in Tencent side.